core/src/memory.rs

```diff
@@ -874,6 +874,14 @@ mod tests {
                         allocations(u32::from(static_pages) as u16, end_exclusive as u16),
                     )
                 })
+                .prop_map(
+                    |(max_pages, mem_size, static_pages, allocations)| MemoryParams {
+                        max_pages,
+                        mem_size,
+                        static_pages,
+                        allocations,
+                    },
+                )
```

Comment:
It's not possible to make this mapping inside the `prop_flat_map` call because it operates with strategies and not values, which these strategies map to.